projects/libjpeg-turbo/build.sh

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 D. R. Commander
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+set -e
+set -u
+
+for branch in main 2.0.x dev; do
+	if [ "$branch" = "dev" -a ! -d libjpeg-turbo.$branch ]; then
+		continue
+	fi
+	pushd libjpeg-turbo.$branch
+	if [ "$branch" = "main" ]; then
+		sh fuzz/build.sh
+	else
+		sh fuzz/build.sh _$branch
```

Comment:
Just double checking: This adds a suffix to the target names right?

Reply:
Yes